install/unifi-install.sh

```diff
@@ -29,10 +29,29 @@ $STD apt-get install -y temurin-17-jre
 msg_ok "Installed Eclipse Temurin JRE"
 
 msg_info "Installing MongoDB"
-wget -qO- https://www.mongodb.org/static/pgp/server-7.0.asc | gpg --dearmor >/usr/share/keyrings/mongodb-server-7.0.gpg
-echo "deb [ signed-by=/usr/share/keyrings/mongodb-server-7.0.gpg ] http://repo.mongodb.org/apt/debian bookworm/mongodb-org/7.0 main" >/etc/apt/sources.list.d/mongodb-org-7.0.list
-$STD apt-get update
-$STD apt-get install -y mongodb-org
+if [ "$MONGO_VERSION" = "4.2" ]; then
+  # Check for libssl1.1 and install if missing
+  if ! dpkg -l | grep -q "libssl1.1"; then
+    msg_info "libssl1.1 not found. Installing..."
+    wget http://security.debian.org/debian-security/pool/updates/main/o/openssl/libssl1.1_1.1.1n-0+deb10u6_amd64.deb
+    dpkg -i libssl1.1_1.1.1n-0+deb10u6_amd64.deb
+    apt-get install -f -y  # Fix any broken dependencies
```

Comment:
```suggestion
    $STD  apt-get install -f -y  # Fix any broken dependencies
```

Reply:
Why fix broken Install?

Reply:
That fix broken doesn't fix broken installs, it fixes broken dependencies, since i'm installing a library that is not commonly found in the standard build of Debian 12 there maybe other older library sets that libssl1.1 depends on so that flag tells apt-get to find and install any missing dependencies.